modules/ztp-create-factory-hub-cluster.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="preparing-the-factory-install-environment_{context}"]
+= Preparing the factory install environment
+
+.Prerequisites
+Before you can provision edge clusters at scale, you must meet the following prerequisites:
+
+=== Base
+
+* Deploy the {product-title} cluster with three control plane nodes following the guidance in the section "Deploying installer-provisioned clusters on bare metal". Alternatively you can use the technology preview Assisted Installer from link:https://cloud.redhat.com/[cloud.redhat.com] to create the cluster.
+** All Cluster Operators are available.
+** Cluster is reachable using a `KUBECONFIG` file.
+** The API, API-INT and ingress should be deployed on the DHCP external Network (factory network).
+
+=== Storage
+
+* Storage can be provided by installing the Local Storage Operator and by using local volumes. However it is recommended to use OpenShift Data Foundation. Deploy the OpenShift Data Foundation following the guidance in link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/deploying_openshift_data_foundation_using_bare_metal_infrastructure/index[Deploying OpenShift Data Foundation using bare metal infrastructure].
+
+** Create storage classes and persistent volumes with at least 200Gb of storage (NVMe or SSD) for:
+
+    *** 2 persistent volumes for Assisted Installer.
+    *** 1 for the hub internal registry that is for the mirror of the images. At least 200Gb is required on the hub, more may be required if ODF is installed.
+    *** 1 persistent volume for HTTPD that hosts the Red Hat Enterprise Linux CoreOS (RHCOS) images.
+    *** 1 for zero touch provisioning factory workflows (ZTPFW).
```

Comment:
1 storage class or 1 persistent volume?  I assume PV... should be made clear on each line where we say 
"1 for..."